### PR TITLE
Create Wallet_ETH.sol

### PR DESCRIPTION
**Pull request by cat-35**
We need this to be in main, @aramservices198. Can you do it?

### CODE REVIEW

File: Wallet_ETH.sol [hunk 1]
```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.5.1;
+contract Wallet_ETH {
+    mapping(address => uint256) public balances;
+    address payable wallet;
+    event Purchase(address indexed _buyer, uint256 _amount);
+    constructor(address payable _wallet) public {
```
Comment (aramservices198) [inline Wallet_ETH.sol:6]:
```suggestion
    constructor(address payable _aram-services_wallet_platform;) public {
```

File: Wallet_ETH.sol [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        wallet = _wallet;
+    }
+    function buyToken() public payable{
```
Comment (aramservices198) [inline Wallet_ETH.sol:9]:
```suggestion
    function buyETH() public payable{
```
or
```suggestion
    function buyEthereum() public payable{
```

**Review by aramservices198 — CHANGES_REQUESTED** [review]

**Review by cat-35 — COMMENTED** [review]
Thanks!

**Comment by cat-35** [comment]
Merged!

**Pull request merged**